tests/attr/test_deeplift_basic.py

```diff
@@ -263,8 +275,13 @@ def test_lin_maxpool_lin_classification(self):
         assertArraysAlmostEqual(delta.detach().numpy(), expected_delta)
 
     def _deeplift_assert(
-        self, model, attr_method, inputs, baselines, custom_attr_func=None
-    ):
+        self,
+        model: Module,
+        attr_method: Union[DeepLift, DeepLiftShap],
+        inputs: Tuple[Tensor, ...],
+        baselines,  # ?
```

Comment:
You can leave out type hinting this argument, it's a little messy since it's used by both DeepLift and DeepLiftShap, but they don't have consistent baseline arguments.